src/shared/ExtensionMessage.ts

```diff
@@ -192,6 +193,8 @@ export interface ClineSayTool {
 	filePattern?: string
 	mode?: string
 	reason?: string
+	text?: string
+	task?: string
```

Comment:
Maybe could just use the existing `content`? Not sure we need to add these two.

Reply:
Agreed.  Fixed.